sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/util/TelemetryHelper.java

```diff
@@ -96,7 +96,7 @@ private static OpenTelemetry init() {
         }
         AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
 
-        AzureMonitor.customize(sdkBuilder, applicationInsightsConnectionString);
+        AzureMonitorExporter.customize(sdkBuilder, applicationInsightsConnectionString);
```

Comment:
please rename `applicationInsightsConnectionString` to 'connectionString`.

Reply:
This class belongs to the `azure-messaging-eventhubs-stress` project.  I would prefer not to change the name chosen by an owner of this project

Reply:
it's mine :) 
I'll prefer to keep it since there could be multiple connectionStrings for multiple services in the context of EventHubs tests. 

Thanks for updating it!